src/style-spec/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 13.10.0
+
+### ✨ Features and improvements
```

Comment:
The style spec changelog doesn’t have the tweaks made to the 1.6.0 changelog. If there is an opportunity to edit it, I think we should re-copy those over

Reply:
done in 8424208